src/common/components/MainNav/MainNav.module.scss

```diff
@@ -0,0 +1,17 @@
+.mainnav {
```

Comment:
Since your component name has two words (Main + Nav), you may want to rename this CSS class to `.main-nav` for readability :)